crates/polars-plan/src/dsl/arithmetic.rs

```diff
@@ -43,6 +45,15 @@ impl Rem for Expr {
     }
 }
 
+#[cfg(feature = "negate")]
+impl Neg for Expr {
```

Comment:
I implemented this as a trait to facilitate `-col(...)`. However, using `col(...).neg()` now requires the user to importing the `Neg` trait. Not sure if this is desirable.

Reply:
We can add `std::ops::Neg` to the prelude.

Reply:
I figured as much, but I couldn't find the other `std::ops` in any preludes - maybe I didn't look hard enough. I'll add it!